WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -190,6 +183,9 @@ extension CardReaderSettingsConnectView: UITableViewDelegate {
         if row == .connectHelpHintChargeReader || row == .connectHelpHintTurnOnReader || row == .connectHelpHintEnableBluetooth {
             return 70
         }
+        if row == .connectLearnMore {
```

Comment:
I didn't want to group the unrelated views with each other is all. Can change.

Reply:
Changed to be a property of the Row Enum in https://github.com/woocommerce/woocommerce-ios/pull/3761/commits/2022ea4b09e87cc0c85f6d122cdde68746dd2406
This will also allow `heightForRowAt` to move into a superclass in a follow on PR.